// @frameless/design-tokens/dist/index.js
/**
 * Do not edit directly
 * Generated on Mon, 17 Jun 2024 20:44:11 GMT
 */

export const utrechtUnorderedListColor = "#122529";
export const utrechtUnorderedListMarkerBorderColor = "#122529";
export const utrechtUnorderedListMarkerColor = "#122529";
export const utrechtUnorderedListFontFamily = "'Fira Sans', Helvetica, Arial, sans-serif";
export const utrechtUnorderedListFontSize = "18px";
export const utrechtUnorderedListFontWeight = 400;
export const utrechtUnorderedListLineHeight = "150%";
export const utrechtUnorderedListPaddingInlineStart = "24px";
export const utrechtUnorderedListItemMarginBlockEnd = "4px";
export const utrechtUnorderedListItemMarginBlockStart = "4px";
export const utrechtTextboxBorderRadius = "0px";
export const utrechtTextboxFontFamily = "'Fira Sans', Helvetica, Arial, sans-serif";
export const utrechtTextboxFontSize = "18px";
export const utrechtTextboxFontWeight = 400;
export const utrechtTextboxLineHeight = "150%";
export const utrechtTextboxMaxInlineSize = "400px";
export const utrechtTextboxPaddingBlockEnd = "12px";
export const utrechtTextboxPaddingBlockStart = "12px";
export const utrechtTextboxPaddingInlineEnd = "12px";
export const utrechtTextboxPaddingInlineStart = "12px";
export const utrechtTextboxBackgroundColor = "#FFFFFF";
export const utrechtTextboxBorderColor = "#484B51";
export const utrechtTextboxColor = "#1F4047";
export const utrechtTextboxInvalidBackgroundColor = "#FFFFFF";
export const utrechtTextboxInvalidBorderColor = "#BA1212";
export const utrechtTextboxInvalidColor = "#1F4047";
export const utrechtTextboxInvalidBorderWidth = "2px";
export const utrechtTextboxPlaceholderColor = "#60646C";
export const utrechtTextboxBorderBlockEndWidth = "auto";
export const utrechtTextboxBorderWidth = "1px";
export const utrechtTextboxFocusBorderWidth = "2px";
export const utrechtTextboxFocusBackgroundColor = "#FFFFFF";
export const utrechtTextboxFocusBorderColor = "#122529";
export const utrechtTextboxFocusColor = "#39051F";
export const utrechtTextboxDisabledBackgroundColor = "#C9CBCF";
export const utrechtTextboxDisabledBorderColor = "#484B51";
export const utrechtTextboxDisabledColor = "#303236";
export const utrechtTextboxReadOnlyBackgroundColor = "#C9CBCF";
export const utrechtTextboxReadOnlyBorderColor = "#484B51";
export const utrechtTextboxReadOnlyColor = "#303236";
export const utrechtTextboxHoverBorderWidth = "2px";
export const utrechtTextboxHoverBackgroundColor = "#E4E5E7";
export const utrechtTextboxHoverBorderColor = "#303236";
export const utrechtTextboxHoverColor = "#122529";
export const utrechtTextareaMaxInlineSize = "400px";
export const utrechtTextareaPaddingBlockEnd = "12px";
export const utrechtTextareaPaddingBlockStart = "12px";
export const utrechtTextareaPaddingInlineEnd = "12px";
export const utrechtTextareaPaddingInlineStart = "12px";
export const utrechtTextareaBackgroundColor = "#FFFFFF";
export const utrechtTextareaBorderColor = "#484B51";
export const utrechtTextareaColor = "#1F4047";
export const utrechtTextareaInvalidBackgroundColor = "#FFFFFF";
export const utrechtTextareaInvalidBorderColor = "#BA1212";
export const utrechtTextareaInvalidColor = "#1F4047";
export const utrechtTextareaInvalidBorderWidth = "2px";
export const utrechtTextareaPlaceholderColor = "#60646C";
export const utrechtTextareaFocusBackgroundColor = "#FFFFFF";
export const utrechtTextareaFocusBorderColor = "#122529";
export const utrechtTextareaFocusColor = "#39051F";
export const utrechtTextareaFocusBorderWidth = "2px";
export const utrechtTextareaDisabledBackgroundColor = "#C9CBCF";
export const utrechtTextareaDisabledBorderColor = "#484B51";
export const utrechtTextareaDisabledColor = "#303236";
export const utrechtTextareaReadOnlyBackgroundColor = "#C9CBCF";
export const utrechtTextareaReadOnlyBorderColor = "#484B51";
export const utrechtTextareaReadOnlyColor = "#303236";
export const utrechtTextareaHoverBackgroundColor = "#E4E5E7";
export const utrechtTextareaHoverBorderColor = "#303236";
export const utrechtTextareaHoverColor = "#122529";
export const utrechtTextareaHoverBorderWidth = "2px";
export const utrechtTextareaBorderRadius = "0px";
export const utrechtTextareaBorderBottomWidth = "auto";
export const utrechtTextareaBorderWidth = "1px";
export const utrechtTextareaFontFamily = "'Fira Sans', Helvetica, Arial, sans-serif";
export const utrechtTextareaFontWeight = 400;
export const utrechtTextareaLineHeight = "150%";
export const utrechtTextareaFontSize = "18px";
export const utrechtTableHeaderCellLineHeight = "150%";
export const utrechtTableHeaderCellColor = "#122529";
export const utrechtTableHeaderCellFontFamily = "'Fira Sans', Helvetica, Arial, sans-serif";
export const utrechtTableHeaderCellFontWeight = 600;
export const utrechtTableHeaderCellFontSize = "18px";
export const utrechtTableCaptionLineHeight = "125%";
export const utrechtTableCaptionMarginBlockEnd = "24px";
export const utrechtTableCaptionColor = "#122529";
export const utrechtTableCaptionFontFamily = "'Fira Sans', Helvetica, Arial, sans-serif";
export const utrechtTableCaptionFontWeight = 600;
export const utrechtTableCaptionFontSize = "24px";
export const utrechtTableCellPaddingBlockEnd = "12px";
export const utrechtTableCellPaddingBlockStart = "12px";
export const utrechtTableCellPaddingInlineEnd = "16px";
export const utrechtTableCellPaddingInlineStart = "16px";
export const utrechtTableDataCellColor = "#122529";
export const utrechtTableDataCellFontFamily = "'Fira Sans', Helvetica, Arial, sans-serif";
export const utrechtTableDataCellFontWeight = 400;
export const utrechtTableDataCellLineHeight = "150%";
export const utrechtTableDataCellFontSize = "18px";
export const utrechtTableHeaderBorderBlockEndWidth = "2px";
export const utrechtTableHeaderBorderBlockEndColor = "#122529";
export const utrechtTableHeaderBackgroundColor = "transparent";
export const utrechtTableFooterBorderBlockEndWidth = "1px";
export const utrechtTableFooterBorderBlockEndColor = "#122529";
export const utrechtTableFooterBackgroundColor = "transparent";
export const utrechtTableRowBorderBlockEndWidth = "1px";
export const utrechtTableRowBorderBlockEndColor = "#122529";
export const utrechtTableRowBackgroundColor = "transparent";
export const utrechtTableFooterCellFontWeight = 600;
export const utrechtTableFooterCellFontSize = "18px";
export const utrechtTableFooterCellColor = "#122529";
export const utrechtTableFooterCellFontFamily = "'Fira Sans', Helvetica, Arial, sans-serif";
export const utrechtTableFooterCellLineHeight = "150%";
export const utrechtTableContainerBoxInlineEndShadow = {"x":0,"y":16,"blur":48,"spread":0,"color":"#E4E5E7","type":"dropShadow"};
export const utrechtTableContainerBoxInlineStartShadow = {"x":0,"y":16,"blur":48,"spread":0,"color":"#E4E5E7","type":"dropShadow"};
export const utrechtStatusBadgeFontSize = "16px";
export const utrechtStatusBadgeLineHeight = "150%";
export const utrechtStatusBadgeFontWeight = 600;
export const utrechtStatusBadgeBorderWidth = "1px";
export const utrechtStatusBadgeFontFamily = "'Fira Sans', Helvetica, Arial, sans-serif";
export const utrechtStatusBadgeBorderRadius = "4px";
export const utrechtStatusBadgePaddingInline = "4px";
export const utrechtStatusBadgePaddingBlock = "2px";
export const utrechtStatusBadgeInformativeBackgroundColor = "#D4F2F7";
export const utrechtStatusBadgeInformativeBorderColor = "transparent";
export const utrechtStatusBadgeInformativeColor = "#1F4047";
export const utrechtStatusBadgePositiveBackgroundColor = "#DDF8E4";
export const utrechtStatusBadgePositiveBorderColor = "transparent";
export const utrechtStatusBadgePositiveColor = "#0D4A1E";
export const utrechtStatusBadgeNegativeBackgroundColor = "#FDECEC";
export const utrechtStatusBadgeNegativeBorderColor = "transparent";
export const utrechtStatusBadgeNegativeColor = "#920707";
export const utrechtStatusBadgeWarningBackgroundColor = "#FDEDD8";
export const utrechtStatusBadgeWarningBorderColor = "transparent";
export const utrechtStatusBadgeWarningColor = "#764504";
export const utrechtStatusBadgeBackgroundColor = "#1F4047";
export const utrechtStatusBadgeBorderColor = "transparent";
export const utrechtStatusBadgeColor = "#FFFFFF";
export const utrechtSkipLinkFontWeight = 600;
export const utrechtSkipLinkFontFamily = "'Fira Sans', Helvetica, Arial, sans-serif";
export const utrechtSkipLinkFontSize = "18px";
export const utrechtSkipLinkLineHeight = "150%";
export const utrechtSkipLinkMinBlockSize = "48px";
export const utrechtSkipLinkMinInlineSize = "48px";
export const utrechtSkipLinkPaddingBlockEnd = "12px";
export const utrechtSkipLinkPaddingBlockStart = "12px";
export const utrechtSkipLinkPaddingInlineEnd = "16px";
export const utrechtSkipLinkPaddingInlineStart = "16px";
export const utrechtSkipLinkFocusBackgroundColor = "#9DFF40";
export const utrechtSkipLinkFocusBorderColor = "transparent";
export const utrechtSkipLinkFocusColor = "#000000";
export const utrechtSkipLinkFocusTextDecoration = "none";
export const utrechtSkipLinkBackgroundColor = "#122529";
export const utrechtSkipLinkBorderColor = "transparent";
export const utrechtSkipLinkColor = "#FFFFFF";
export const utrechtSkipLinkBorderWidth = "1px";
export const utrechtSkipLinkBoxBlockEndShadow = {"x":0,"y":16,"blur":48,"spread":0,"color":"#E4E5E7","type":"dropShadow"};
export const utrechtSkipLinkTextDecoration = "underline";
export const utrechtSeparatorColor = "#122529";
export const utrechtSeparatorBlockSize = "4px";
export const utrechtSeparatorPaddingBlockEnd = "12px";
export const utrechtSeparatorPaddingBlockStart = "12px";
export const utrechtSelectIconSize = "24px";
export const utrechtSelectBackgroundColor = "#FFFFFF";
export const utrechtSelectBorderColor = "#484B51";
export const utrechtSelectColor = "#1F4047";
export const utrechtSelectInvalidBackgroundColor = "#FFFFFF";
export const utrechtSelectInvalidBorderColor = "#BA1212";
export const utrechtSelectInvalidColor = "#1F4047";
export const utrechtSelectInvalidBorderWidth = "2px";
export const utrechtSelectFocusBackgroundColor = "#FFFFFF";
export const utrechtSelectFocusBorderColor = "#122529";
export const utrechtSelectFocusColor = "#39051F";
export const utrechtSelectFocusBorderWidth = "2px";
export const utrechtSelectDisabledBackgroundColor = "#C9CBCF";
export const utrechtSelectDisabledBorderColor = "#484B51";
export const utrechtSelectDisabledColor = "#303236";
export const utrechtSelectHoverBackgroundColor = "#E4E5E7";
export const utrechtSelectHoverBorderColor = "#303236";
export const utrechtSelectHoverColor = "#122529";
export const utrechtSelectHoverBorderWidth = "2px";
export const utrechtSelectPaddingBlockEnd = "12px";
export const utrechtSelectPaddingBlockStart = "12px";
export const utrechtSelectPaddingInlineEnd = "12px";
export const utrechtSelectPaddingInlineStart = "12px";
export const utrechtSelectBorderRadius = "0px";
export const utrechtSelectBorderBottomWidth = "auto";
export const utrechtSelectBorderWidth = "1px";
export const utrechtSelectFontFamily = "'Fira Sans', Helvetica, Arial, sans-serif";
export const utrechtSelectFontWeight = 400;
export const utrechtSelectLineHeight = "150%";
export const utrechtSelectFontSize = "18px";
export const utrechtSelectMaxInlineSize = "400px";
export const utrechtRadioActiveBorderWidth = "2px";
export const utrechtRadioActiveBackgroundColor = "#C9CBCF";
export const utrechtRadioActiveBorderColor = "#484B51";
export const utrechtRadioSize = "24px";
export const utrechtRadioDotSize = "8px";
export const utrechtRadioBackgroundColor = "#FFFFFF";
export const utrechtRadioBorderColor = "#484B51";
export const utrechtRadioInvalidBackgroundColor = "#FFFFFF";
export const utrechtRadioInvalidBorderColor = "#BA1212";
export const utrechtRadioInvalidBorderWidth = "2px";
export const utrechtRadioFocusBackgroundColor = "#FFFFFF";
export const utrechtRadioFocusBorderColor = "#122529";
export const utrechtRadioFocusBorderWidth = "2px";
export const utrechtRadioDisabledBackgroundColor = "#C9CBCF";
export const utrechtRadioDisabledBorderColor = "#484B51";
export const utrechtRadioCheckedBackgroundColor = "#4BA7B2";
export const utrechtRadioCheckedBorderColor = "transparent";
export const utrechtRadioCheckedColor = "#FFFFFF";
export const utrechtRadioCheckedHoverBackgroundColor = "#39051F";
export const utrechtRadioCheckedHoverBorderColor = "transparent";
export const utrechtRadioCheckedHoverColor = "#FFFFFF";
export const utrechtRadioCheckedHoverBorderWidth = "2px";
export const utrechtRadioCheckedActiveBackgroundColor = "#3C8790";
export const utrechtRadioCheckedActiveBorderColor = "transparent";
export const utrechtRadioCheckedActiveColor = "#FFFFFF";
export const utrechtRadioCheckedActiveBorderWidth = "2px";
export const utrechtRadioCheckedFocusBackgroundColor = "#FFFFFF";
export const utrechtRadioCheckedFocusBorderColor = "#122529";
export const utrechtRadioCheckedFocusColor = "#39051F";
export const utrechtRadioCheckedFocusBorderWidth = "2px";
export const utrechtRadioCheckedDisabledBackgroundColor = "#303236";
export const utrechtRadioCheckedDisabledBorderColor = "transparent";
export const utrechtRadioCheckedDisabledColor = "#FFFFFF";
export const utrechtRadioCheckedBorderWidth = "1px";
export const utrechtRadioHoverBackgroundColor = "#E4E5E7";
export const utrechtRadioHoverBorderColor = "#303236";
export const utrechtRadioHoverBorderWidth = "2px";
export const utrechtRadioBorderRadius = "999px";
export const utrechtRadioBorderWidth = "1px";
export const utrechtParagraphColor = "#122529";
export const utrechtParagraphFontFamily = "'Fira Sans', Helvetica, Arial, sans-serif";
export const utrechtParagraphFontSize = "18px";
export const utrechtParagraphFontWeight = 400;
export const utrechtParagraphLineHeight = "150%";
export const utrechtParagraphLeadColor = "#122529";
export const utrechtParagraphLeadFontFamily = "'Fira Sans', Helvetica, Arial, sans-serif";
export const utrechtParagraphLeadFontSize = "20px";
export const utrechtParagraphLeadFontWeight = 400;
export const utrechtParagraphLeadLineHeight = "150%";
export const utrechtParagraphSmallPrintColor = "#122529";
export const utrechtParagraphSmallPrintFontFamily = "'Fira Sans', Helvetica, Arial, sans-serif";
export const utrechtParagraphSmallPrintFontSize = "16px";
export const utrechtParagraphSmallPrintFontWeight = 400;
export const utrechtParagraphSmallPrintLineHeight = "150%";
export const utrechtOrderedListPaddingInlineStart = "24px";
export const utrechtOrderedListColor = "#122529";
export const utrechtOrderedListFontFamily = "'Fira Sans', Helvetica, Arial, sans-serif";
export const utrechtOrderedListFontWeight = 400;
export const utrechtOrderedListFontSize = "18px";
export const utrechtOrderedListLineHeight = "150%";
export const utrechtOrderedListItemMarginBlockEnd = "4px";
export const utrechtOrderedListItemMarginBlockStart = "4px";
export const utrechtLinkLineHeight = "150%";
export const utrechtLinkColor = "#850B48";
export const utrechtLinkTextDecorationColor = "#850B48";
export const utrechtLinkActiveColor = "#55072E";
export const utrechtLinkActiveTextDecoration = "none";
export const utrechtLinkFocusBackgroundColor = "#9DFF40";
export const utrechtLinkFocusColor = "#000000";
export const utrechtLinkFocusTextDecoration = "none";
export const utrechtLinkFocusTextDecorationThickness = "auto";
export const utrechtLinkHoverColor = "#39051F";
export const utrechtLinkHoverTextDecoration = "none";
export const utrechtLinkHoverTextDecorationThickness = "auto";
export const utrechtLinkVisitedColor = "#850B48";
export const utrechtLinkTextDecoration = "underline";
export const utrechtLinkIconSize = "24px";
export const utrechtLinkIconMarginInline = "4px";
export const utrechtLinkFontSize = "18px";
export const utrechtLinkFontFamily = "'Fira Sans', Helvetica, Arial, sans-serif";
export const utrechtLinkTextDecorationThickness = "auto";
export const utrechtLinkTextUnderlineOffset = "auto";
export const utrechtLinkFontWeight = 400;
export const utrechtHeading1Color = "#122529";
export const utrechtHeading1FontFamily = "'Fira Sans', Helvetica, Arial, sans-serif";
export const utrechtHeading1FontWeight = 600;
export const utrechtHeading1LineHeight = "125%";
export const utrechtHeading1FontSize = "40px";
export const utrechtHeading2Color = "#122529";
export const utrechtHeading2FontFamily = "'Fira Sans', Helvetica, Arial, sans-serif";
export const utrechtHeading2FontWeight = 600;
export const utrechtHeading2LineHeight = "125%";
export const utrechtHeading2FontSize = "32px";
export const utrechtHeading3Color = "#122529";
export const utrechtHeading3FontFamily = "'Fira Sans', Helvetica, Arial, sans-serif";
export const utrechtHeading3FontWeight = 600;
export const utrechtHeading3LineHeight = "125%";
export const utrechtHeading3FontSize = "24px";
export const utrechtHeading4Color = "#122529";
export const utrechtHeading4FontFamily = "'Fira Sans', Helvetica, Arial, sans-serif";
export const utrechtHeading4FontWeight = 600;
export const utrechtHeading4LineHeight = "150%";
export const utrechtHeading4FontSize = "20px";
export const utrechtHeading5Color = "#122529";
export const utrechtHeading5FontFamily = "'Fira Sans', Helvetica, Arial, sans-serif";
export const utrechtHeading5FontWeight = 600;
export const utrechtHeading5LineHeight = "150%";
export const utrechtHeading5FontSize = "18px";
export const utrechtHeading6Color = "#122529";
export const utrechtHeading6FontFamily = "'Fira Sans', Helvetica, Arial, sans-serif";
export const utrechtHeading6FontWeight = 600;
export const utrechtHeading6LineHeight = "150%";
export const utrechtHeading6FontSize = "16px";
export const utrechtFormFieldErrorMessageColor = "#920707";
export const utrechtFormFieldErrorMessageFontFamily = "'Fira Sans', Helvetica, Arial, sans-serif";
export const utrechtFormFieldErrorMessageFontSize = "18px";
export const utrechtFormFieldErrorMessageFontWeight = 400;
export const utrechtFormFieldErrorMessageLineHeight = "150%";
export const utrechtFormFieldDescriptionColor = "#787D87";
export const utrechtFormFieldDescriptionFontFamily = "'Fira Sans', Helvetica, Arial, sans-serif";
export const utrechtFormFieldDescriptionFontSize = "18px";
export const utrechtFormFieldDescriptionFontWeight = 400;
export const utrechtFormFieldDescriptionLineHeight = "150%";
export const utrechtFormFieldInvalidBorderInlineStartColor = "#BA1212";
export const utrechtFormFieldInvalidBorderInlineStartWidth = "2px";
export const utrechtFormFieldInvalidPaddingInlineStart = "16px";
export const utrechtDrawerHeaderLabelFontFamily = "'Fira Sans', Helvetica, Arial, sans-serif";
export const utrechtDrawerHeaderLabelFontSize = "18px";
export const utrechtDrawerHeaderLabelFontWeight = 600;
export const utrechtDrawerHeaderLabelLineHeight = "150%";
export const utrechtCounterBadgeFontSize = "16px";
export const utrechtCounterBadgeLineHeight = "150%";
export const utrechtCounterBadgeMinBlockSize = "24px";
export const utrechtCounterBadgeMinInlineSize = "24px";
export const utrechtCounterBadgeFontWeight = 600;
export const utrechtCounterBadgeBorderWidth = "1px";
export const utrechtCounterBadgeBorderRadius = "999px";
export const utrechtCounterBadgeFontFamily = "'Fira Sans', Helvetica, Arial, sans-serif";
export const utrechtCounterBadgeBackgroundColor = "#AAE5EE";
export const utrechtCounterBadgeBorderColor = "#4BA7B2";
export const utrechtCounterBadgeColor = "#122529";
export const utrechtCounterBadgePaddingInline = "8px";
export const utrechtCounterBadgePaddingBlock = "4px";
export const utrechtCheckboxBorderRadius = "0px";
export const utrechtCheckboxSize = "24px";
export const utrechtCheckboxIconSize = "24px";
export const utrechtCheckboxBackgroundColor = "#FFFFFF";
export const utrechtCheckboxBorderColor = "#484B51";
export const utrechtCheckboxInvalidBackgroundColor = "#FFFFFF";
export const utrechtCheckboxInvalidBorderColor = "#BA1212";
export const utrechtCheckboxInvalidBorderWidth = "2px";
export const utrechtCheckboxFocusBackgroundColor = "#FFFFFF";
export const utrechtCheckboxFocusBorderColor = "#122529";
export const utrechtCheckboxFocusBorderWidth = "2px";
export const utrechtCheckboxDisabledBackgroundColor = "#C9CBCF";
export const utrechtCheckboxDisabledBorderColor = "#484B51";
export const utrechtCheckboxCheckedBackgroundColor = "#4BA7B2";
export const utrechtCheckboxCheckedBorderColor = "transparent";
export const utrechtCheckboxCheckedColor = "#FFFFFF";
export const utrechtCheckboxCheckedBorderWidth = "1px";
export const utrechtCheckboxCheckedDisabledBackgroundColor = "#303236";
export const utrechtCheckboxCheckedDisabledBorderColor = "transparent";
export const utrechtCheckboxCheckedDisabledColor = "#FFFFFF";
export const utrechtCheckboxCheckedFocusBorderWidth = "2px";
export const utrechtCheckboxCheckedFocusBackgroundColor = "#FFFFFF";
export const utrechtCheckboxCheckedFocusBorderColor = "#122529";
export const utrechtCheckboxCheckedFocusColor = "#39051F";
export const utrechtCheckboxCheckedHoverBorderWidth = "2px";
export const utrechtCheckboxCheckedHoverBackgroundColor = "#39051F";
export const utrechtCheckboxCheckedHoverBorderColor = "transparent";
export const utrechtCheckboxCheckedHoverColor = "#FFFFFF";
export const utrechtCheckboxCheckedActiveBorderWidth = "2px";
export const utrechtCheckboxCheckedActiveBackgroundColor = "#3C8790";
export const utrechtCheckboxCheckedActiveBorderColor = "transparent";
export const utrechtCheckboxCheckedActiveColor = "#FFFFFF";
export const utrechtCheckboxIndeterminateBackgroundColor = "#4BA7B2";
export const utrechtCheckboxIndeterminateBorderColor = "transparent";
export const utrechtCheckboxIndeterminateColor = "#FFFFFF";
export const utrechtCheckboxIndeterminateBorderWidth = "1px";
export const utrechtCheckboxIndeterminateDisabledBackgroundColor = "#303236";
export const utrechtCheckboxIndeterminateDisabledBorderColor = "transparent";
export const utrechtCheckboxIndeterminateDisabledColor = "#FFFFFF";
export const utrechtCheckboxIndeterminateActiveBackgroundColor = "#3C8790";
export const utrechtCheckboxIndeterminateActiveBorderColor = "transparent";
export const utrechtCheckboxIndeterminateActiveColor = "#FFFFFF";
export const utrechtCheckboxIndeterminateActiveBorderWidth = "2px";
export const utrechtCheckboxIndeterminateHoverBackgroundColor = "#39051F";
export const utrechtCheckboxIndeterminateHoverBorderColor = "transparent";
export const utrechtCheckboxIndeterminateHoverColor = "#FFFFFF";
export const utrechtCheckboxIndeterminateHoverBorderWidth = "2px";
export const utrechtCheckboxIndeterminateFocusBackgroundColor = "#FFFFFF";
export const utrechtCheckboxIndeterminateFocusBorderColor = "#122529";
export const utrechtCheckboxIndeterminateFocusColor = "#39051F";
export const utrechtCheckboxIndeterminateFocusBorderWidth = "2px";
export const utrechtCheckboxBorderWidth = "1px";
export const utrechtCheckboxHoverBorderWidth = "2px";
export const utrechtCheckboxHoverBackgroundColor = "#E4E5E7";
export const utrechtCheckboxHoverBorderColor = "#303236";
export const utrechtCheckboxActiveBorderWidth = "2px";
export const utrechtCheckboxActiveBackgroundColor = "#C9CBCF";
export const utrechtCheckboxActiveBorderColor = "#484B51";
export const utrechtButtonGroupBackgroundColor = "transparent";
export const utrechtButtonGroupColumnGap = "8px";
export const utrechtButtonGroupMarginBlockEnd = "0px";
export const utrechtButtonGroupMarginBlockStart = "0px";
export const utrechtButtonGroupPaddingBlockEnd = "0px";
export const utrechtButtonGroupPaddingBlockStart = "0px";
export const utrechtButtonGroupRowGap = "8px";
export const utrechtButtonPrimaryActionHoverBackgroundColor = "#39051F";
export const utrechtButtonPrimaryActionHoverBorderColor = "transparent";
export const utrechtButtonPrimaryActionHoverColor = "#FFFFFF";
export const utrechtButtonPrimaryActionBackgroundColor = "#850B48";
export const utrechtButtonPrimaryActionBorderColor = "transparent";
export const utrechtButtonPrimaryActionColor = "#FFFFFF";
export const utrechtButtonPrimaryActionDisabledBackgroundColor = "#C9CBCF";
export const utrechtButtonPrimaryActionDisabledBorderColor = "transparent";
export const utrechtButtonPrimaryActionDisabledColor = "#93979F";
export const utrechtButtonPrimaryActionFocusBackgroundColor = "#9DFF40";
export const utrechtButtonPrimaryActionFocusBorderColor = "transparent";
export const utrechtButtonPrimaryActionFocusColor = "#000000";
export const utrechtButtonPrimaryActionFontWeight = 600;
export const utrechtButtonPrimaryActionActiveBackgroundColor = "#55072E";
export const utrechtButtonPrimaryActionActiveBorderColor = "transparent";
export const utrechtButtonPrimaryActionActiveColor = "#FFFFFF";
export const utrechtButtonSecondaryActionHoverBackgroundColor = "#E4E5E7";
export const utrechtButtonSecondaryActionHoverBorderColor = "#39051F";
export const utrechtButtonSecondaryActionHoverColor = "#39051F";
export const utrechtButtonSecondaryActionBackgroundColor = "#FFFFFF";
export const utrechtButtonSecondaryActionBorderColor = "#850B48";
export const utrechtButtonSecondaryActionColor = "#850B48";
export const utrechtButtonSecondaryActionDisabledBackgroundColor = "#E4E5E7";
export const utrechtButtonSecondaryActionDisabledBorderColor = "transparent";
export const utrechtButtonSecondaryActionDisabledColor = "#93979F";
export const utrechtButtonSecondaryActionFocusBackgroundColor = "#9DFF40";
export const utrechtButtonSecondaryActionFocusBorderColor = "transparent";
export const utrechtButtonSecondaryActionFocusColor = "#000000";
export const utrechtButtonSecondaryActionFontWeight = 600;
export const utrechtButtonSecondaryActionActiveBackgroundColor = "#E4E5E7";
export const utrechtButtonSecondaryActionActiveBorderColor = "#55072E";
export const utrechtButtonSecondaryActionActiveColor = "#55072E";
export const utrechtButtonBackgroundColor = "transparent";
export const utrechtButtonBorderColor = "#850B48";
export const utrechtButtonColor = "#850B48";
export const utrechtButtonBorderRadius = "8px";
export const utrechtButtonBorderWidth = "1px";
export const utrechtButtonIconSize = "24px";
export const utrechtButtonIconMarginInline = "4px";
export const utrechtButtonFontFamily = "'Fira Sans', Helvetica, Arial, sans-serif";
export const utrechtButtonFontSize = "18px";
export const utrechtButtonFontWeight = 600;
export const utrechtButtonSubtleFontWeight = 600;
export const utrechtButtonSubtleHoverBackgroundColor = "transparent";
export const utrechtButtonSubtleHoverBorderColor = "transparent";
export const utrechtButtonSubtleHoverColor = "#39051F";
export const utrechtButtonSubtleBackgroundColor = "transparent";
export const utrechtButtonSubtleBorderColor = "transparent";
export const utrechtButtonSubtleColor = "#850B48";
export const utrechtButtonSubtleDisabledBackgroundColor = "#C9CBCF";
export const utrechtButtonSubtleDisabledBorderColor = "transparent";
export const utrechtButtonSubtleDisabledColor = "#303236";
export const utrechtButtonSubtleFocusBackgroundColor = "#9DFF40";
export const utrechtButtonSubtleFocusBorderColor = "transparent";
export const utrechtButtonSubtleFocusColor = "#000000";
export const utrechtButtonSubtleActiveBackgroundColor = "#FFFFFF";
export const utrechtButtonSubtleActiveBorderColor = "transparent";
export const utrechtButtonSubtleActiveColor = "#55072E";
export const utrechtButtonLineHeight = "150%";
export const utrechtButtonPaddingBlockEnd = "12px";
export const utrechtButtonPaddingBlockStart = "12px";
export const utrechtButtonPaddingInlineEnd = "16px";
export const utrechtButtonPaddingInlineStart = "16px";
export const utrechtButtonDisabledBackgroundColor = "#C9CBCF";
export const utrechtButtonDisabledBorderColor = "transparent";
export const utrechtButtonDisabledColor = "#303236";
export const utrechtButtonFocusBackgroundColor = "#9DFF40";
export const utrechtButtonFocusBorderColor = "transparent";
export const utrechtButtonFocusColor = "#000000";
export const utrechtButtonHoverBackgroundColor = "#FFFFFF";
export const utrechtButtonHoverBorderColor = "#39051F";
export const utrechtButtonHoverColor = "#39051F";
export const utrechtButtonActiveBackgroundColor = "#FFFFFF";
export const utrechtButtonActiveBorderColor = "#55072E";
export const utrechtButtonActiveColor = "#55072E";
export const utrechtButtonMinBlockSize = "48px";
export const utrechtButtonMinInlineSize = "48px";
export const utrechtBlockquoteAttributionFontSize = "16px";
export const utrechtBlockquoteAttributionFontFamily = "'Fira Sans', Helvetica, Arial, sans-serif";
export const utrechtBlockquoteAttributionLineHeight = "150%";
export const utrechtBlockquoteAttributionFontWeight = 400;
export const utrechtBlockquoteAttributionPaddingBlockStart = "12px";
export const utrechtBlockquoteAttributionColor = "#787D87";
export const utrechtBlockquoteContentFontFamily = "'Fira Sans', Helvetica, Arial, sans-serif";
export const utrechtBlockquoteContentLineHeight = "125%";
export const utrechtBlockquoteContentFontWeight = 400;
export const utrechtBlockquoteContentFontSize = "24px";
export const utrechtBlockquoteContentColor = "#122529";
export const utrechtBlockquotePaddingBlockEnd = "12px";
export const utrechtBlockquotePaddingBlockStart = "12px";
export const utrechtBlockquotePaddingInlineEnd = "0px";
export const utrechtBlockquotePaddingInlineStart = "0px";
export const utrechtBlockquoteBackgroundColor = "transparent";
export const utrechtAlertHeadingFontFamily = "'Fira Sans', Helvetica, Arial, sans-serif";
export const utrechtAlertHeadingFontWeight = 600;
export const utrechtAlertHeadingLineHeight = "125%";
export const utrechtAlertHeadingFontSize = "24px";
export const utrechtAlertColumnGap = "8px";
export const utrechtAlertBorderWidth = "2px";
export const utrechtAlertPaddingBlockEnd = "12px";
export const utrechtAlertPaddingBlockStart = "12px";
export const utrechtAlertPaddingInlineEnd = "16px";
export const utrechtAlertPaddingInlineStart = "16px";
export const utrechtAlertInformativeBackgroundColor = "#D4F2F7";
export const utrechtAlertInformativeBorderColor = "#3C8790";
export const utrechtAlertInformativeColor = "#122529";
export const utrechtAlertIconInformativeColor = "#1F4047";
export const utrechtAlertIconPaddingBlockEnd = "12px";
export const utrechtAlertIconPaddingBlockStart = "12px";
export const utrechtAlertIconPaddingInlineEnd = "0px";
export const utrechtAlertIconPaddingInlineStart = "0px";
export const utrechtAlertIconPositiveColor = "#0D4A1E";
export const utrechtAlertIconNegativeColor = "#920707";
export const utrechtAlertIconWarningColor = "#764504";
export const utrechtAlertIconSize = "24px";
export const utrechtAlertNegativeBackgroundColor = "#FDECEC";
export const utrechtAlertNegativeBorderColor = "#BA1212";
export const utrechtAlertNegativeColor = "#122529";
export const utrechtAlertPositiveBackgroundColor = "#DDF8E4";
export const utrechtAlertPositiveBorderColor = "#16822D";
export const utrechtAlertPositiveColor = "#122529";
export const utrechtAlertWarningBackgroundColor = "#FDEDD8";
export const utrechtAlertWarningBorderColor = "#D57B07";
export const utrechtAlertWarningColor = "#122529";
export const utrechtAlertMessageRowGap = "8px";
export const utrechtAlertMessagePaddingBlockEnd = "12px";
export const utrechtAlertMessagePaddingBlockStart = "8px";
export const utrechtAlertContentRowGap = "8px";
export const utrechtAccordionBorderRadius = "0px";
export const utrechtAccordionSectionBorderBlockEndWidth = "1px";
export const utrechtAccordionSectionBorderWidth = "auto";
export const utrechtAccordionSectionBorderColor = "#122529";
export const utrechtAccordionBodyBorderColor = "transparent";
export const utrechtAccordionBodyBorderWidth = "4px";
export const utrechtAccordionBodyPaddingBlockEnd = "24px";
export const utrechtAccordionBodyPaddingBlockStart = "24px";
export const utrechtAccordionBodyPaddingInlineEnd = "16px";
export const utrechtAccordionBodyPaddingInlineStart = "16px";
export const utrechtAccordionButtonFontSize = "18px";
export const utrechtAccordionButtonLineHeight = "150%";
export const utrechtAccordionButtonIconMarginInline = "8px";
export const utrechtAccordionButtonIconSize = "24px";
export const utrechtAccordionButtonPaddingBlockEnd = "12px";
export const utrechtAccordionButtonPaddingBlockStart = "12px";
export const utrechtAccordionButtonPaddingInlineEnd = "16px";
export const utrechtAccordionButtonPaddingInlineStart = "16px";
export const utrechtAccordionButtonHoverBackgroundColor = "#FFFFFF";
export const utrechtAccordionButtonHoverBorderColor = "transparent";
export const utrechtAccordionButtonHoverColor = "#39051F";
export const utrechtAccordionButtonExpandedBackgroundColor = "#FFFFFF";
export const utrechtAccordionButtonExpandedBorderColor = "transparent";
export const utrechtAccordionButtonExpandedColor = "#850B48";
export const utrechtAccordionButtonBackgroundColor = "#FFFFFF";
export const utrechtAccordionButtonBorderColor = "transparent";
export const utrechtAccordionButtonColor = "#850B48";
export const utrechtAccordionButtonFocusBackgroundColor = "#9DFF40";
export const utrechtAccordionButtonFocusBorderColor = "transparent";
export const utrechtAccordionButtonFocusColor = "#000000";
export const utrechtAccordionButtonActiveBackgroundColor = "#FFFFFF";
export const utrechtAccordionButtonActiveBorderColor = "transparent";
export const utrechtAccordionButtonActiveColor = "#55072E";
export const utrechtAccordionButtonBorderWidth = "1px";
export const utrechtAccordionButtonFontFamily = "'Fira Sans', Helvetica, Arial, sans-serif";
export const utrechtAccordionButtonFontWeight = 600;
export const utrechtRootBackgroundColor = "#FFFFFF";
export const utrechtDocumentBackgroundColor = "#FFFFFF";
export const utrechtDocumentColor = "#122529";
export const utrechtDocumentInverseBackgroundColor = "#122529";
export const utrechtDocumentInverseColor = "#FFFFFF";
export const utrechtDocumentSubtleColor = "#787D87";
export const utrechtDocumentFontFamily = "'Fira Sans', Helvetica, Arial, sans-serif";
export const utrechtDocumentFontSize = "18px";
export const utrechtDocumentFontWeight = 400;
export const utrechtDocumentStrongFontWeight = 600;
export const utrechtDocumentLineHeight = "150%";
export const utrechtHeadingColor = "#122529";
export const utrechtHeadingFontFamily = "'Fira Sans', Helvetica, Arial, sans-serif";
export const utrechtHeadingFontWeight = 600;
export const utrechtContainerBorderColor = "#122529";
export const utrechtContainerBorderWidth = "1px";
export const utrechtLineBorderColor = "#122529";
export const utrechtLineBorderWidth = "1px";
export const utrechtInteractionColor = "#850B48";
export const utrechtInteractionActiveColor = "#55072E";
export const utrechtInteractionActiveInverseColor = "#FFFFFF";
export const utrechtInteractionHoverColor = "#39051F";
export const utrechtInteractionHoverInverseColor = "#FFFFFF";
export const utrechtInteractionInverseColor = "#FFFFFF";
export const utrechtFocusBackgroundColor = "#9DFF40";
export const utrechtFocusColor = "#000000";
export const utrechtFocusOutlineColor = "#000000";
export const utrechtFocusInverseOutlineColor = "#9DFF40";
export const utrechtFocusOutlineOffset = 0;
export const utrechtFocusOutlineStyle = "dotted";
export const utrechtFocusOutlineWidth = "2px";
export const utrechtFormControlAccentColor = "#4BA7B2";
export const utrechtFormControlBackgroundColor = "#FFFFFF";
export const utrechtFormControlBorderColor = "#484B51";
export const utrechtFormControlBorderWidth = "1px";
export const utrechtFormControlColor = "#1F4047";
export const utrechtFormControlActiveAccentColor = "#3C8790";
export const utrechtFormControlActiveBackgroundColor = "#C9CBCF";
export const utrechtFormControlActiveBorderColor = "#484B51";
export const utrechtFormControlActiveBorderWidth = "2px";
export const utrechtFormControlActiveColor = "#122529";
export const utrechtFormControlDisabledAccentColor = "#303236";
export const utrechtFormControlDisabledBackgroundColor = "#C9CBCF";
export const utrechtFormControlDisabledBorderColor = "#484B51";
export const utrechtFormControlDisabledColor = "#303236";
export const utrechtFormControlFocusBackgroundColor = "#FFFFFF";
export const utrechtFormControlFocusBorderColor = "#122529";
export const utrechtFormControlFocusBorderWidth = "2px";
export const utrechtFormControlFocusColor = "#39051F";
export const utrechtFormControlHoverAccentColor = "#39051F";
export const utrechtFormControlHoverBackgroundColor = "#E4E5E7";
export const utrechtFormControlHoverBorderColor = "#303236";
export const utrechtFormControlHoverBorderWidth = "2px";
export const utrechtFormControlHoverColor = "#122529";
export const utrechtFormControlInvalidBackgroundColor = "#FFFFFF";
export const utrechtFormControlInvalidBorderColor = "#BA1212";
export const utrechtFormControlInvalidBorderWidth = "2px";
export const utrechtFormControlInvalidColor = "#920707";
export const utrechtFormControlReadOnlyBackgroundColor = "#C9CBCF";
export const utrechtFormControlReadOnlyBorderColor = "#484B51";
export const utrechtFormControlReadOnlyColor = "#303236";
export const utrechtFormControlPlaceholderColor = "#60646C";
export const utrechtPointerTargetMinSize = "48px";
export const utrechtIconFunctionalSize = "24px";
export const utrechtIconToptaskSize = "48px";
export const utrechtFeedbackInformativeBackgroundColor = "#D4F2F7";
export const utrechtFeedbackInformativeBorderColor = "#3C8790";
export const utrechtFeedbackInformativeColor = "#1F4047";
export const utrechtFeedbackNegativeBackgroundColor = "#FDECEC";
export const utrechtFeedbackNegativeBorderColor = "#BA1212";
export const utrechtFeedbackNegativeColor = "#920707";
export const utrechtFeedbackPositiveBackgroundColor = "#DDF8E4";
export const utrechtFeedbackPositiveBorderColor = "#16822D";
export const utrechtFeedbackPositiveColor = "#0D4A1E";
export const utrechtFeedbackWarningBackgroundColor = "#FDEDD8";
export const utrechtFeedbackWarningBorderColor = "#D57B07";
export const utrechtFeedbackWarningColor = "#764504";
export const utrechtPageMaxInlineSize = "800px";
export const utrechtPageMarginBlockEnd = "32px";
export const utrechtPageMarginBlockStart = "32px";
export const todoToolbarButtonIconMarginInline = "4px";
export const todoToolbarButtonPaddingBlockEnd = "12px";
export const todoToolbarButtonPaddingBlockStart = "12px";
export const todoToolbarButtonPaddingInlineEnd = "8px";
export const todoToolbarButtonPaddingInlineStart = "8px";
export const todoTaskListColor = "#122529";
export const todoTaskListMarkerCheckedColor = "#FFFFFF";
export const todoTaskListMarkerCheckedBackgroundColor = "#16822D";
export const todoTaskListMarkerCheckedBorderColor = "transparent";
export const todoTaskListMarkerSize = "24px";
export const todoTaskListMarkerIconSize = "16px";
export const todoTaskListMarkerBorderWidth = "1px";
export const todoTaskListMarkerNotCheckedBackgroundColor = "#C9CBCF";
export const todoTaskListMarkerNotCheckedBorderColor = "#484B51";
export const todoTaskListMarkerBorderRadius = "0px";
export const todoTaskListFontFamily = "'Fira Sans', Helvetica, Arial, sans-serif";
export const todoTaskListFontSize = "18px";
export const todoTaskListFontWeight = 400;
export const todoTaskListLineHeight = "150%";
export const todoTaskListItemColumnGap = "12px";
export const todoTaskListItemMarginBlockEnd = "4px";
export const todoTaskListItemMarginBlockStart = "4px";
export const todoTaskListRowGap = "16px";
export const todoRadioGroupPaddingBlockEnd = "8px";
export const todoRadioGroupPaddingBlockStart = "8px";
export const todoRadioGroupRowGap = "16px";
export const todoPaginationRelativeLinkBorderWidth = "1px";
export const todoPaginationRelativeLinkIconMarginInline = "4px";
export const todoPaginationRelativeLinkIconSize = "24px";
export const todoPaginationRelativeLinkPaddingBlockEnd = "12px";
export const todoPaginationRelativeLinkPaddingBlockStart = "12px";
export const todoPaginationRelativeLinkPaddingInlineStart = "0px";
export const todoPaginationRelativeLinkPaddingInlineEnd = "0px";
export const todoPaginationRelativeLinkBackgroundColor = "transparent";
export const todoPaginationRelativeLinkBorderColor = "transparent";
export const todoPaginationRelativeLinkColor = "#850B48";
export const todoPaginationRelativeLinkActiveBackgroundColor = "transparent";
export const todoPaginationRelativeLinkActiveBorderColor = "transparent";
export const todoPaginationRelativeLinkActiveColor = "#55072E";
export const todoPaginationRelativeLinkActiveTextDecoration = "underline";
export const todoPaginationRelativeLinkFocusBackgroundColor = "#9DFF40";
export const todoPaginationRelativeLinkFocusColor = "#000000";
export const todoPaginationRelativeLinkFocusTextDecoration = "none";
export const todoPaginationRelativeLinkHoverBackgroundColor = "transparent";
export const todoPaginationRelativeLinkHoverBorderColor = "transparent";
export const todoPaginationRelativeLinkHoverColor = "#39051F";
export const todoPaginationRelativeLinkHoverTextDecoration = "underline";
export const todoPaginationRelativeLinkTextDecoration = "none";
export const todoPaginationRelativeLinkBorderRadius = "0px";
export const todoPaginationRelativeLinkFontWeight = 400;
export const todoPaginationRelativeLinkMinBlockSize = "48px";
export const todoPaginationRelativeLinkMinInlineSize = "48px";
export const todoPaginationRelativeLinkDisabledBackgroundColor = "transparent";
export const todoPaginationRelativeLinkDisabledBorderColor = "transparent";
export const todoPaginationRelativeLinkDisabledColor = "#93979F";
export const todoPaginationPageLinkBorderWidth = "1px";
export const todoPaginationPageLinkFontWeight = 400;
export const todoPaginationPageLinkCurrentFontWeight = 600;
export const todoPaginationPageLinkCurrentTextDecoration = "none";
export const todoPaginationPageLinkCurrentBackgroundColor = "#E4E5E7";
export const todoPaginationPageLinkCurrentBorderColor = "transparent";
export const todoPaginationPageLinkCurrentColor = "#122529";
export const todoPaginationPageLinkBorderRadius = "0px";
export const todoPaginationPageLinkPaddingBlockEnd = "12px";
export const todoPaginationPageLinkPaddingBlockStart = "12px";
export const todoPaginationPageLinkPaddingInlineEnd = "12px";
export const todoPaginationPageLinkPaddingInlineStart = "12px";
export const todoPaginationPageLinkFocusTextDecoration = "none";
export const todoPaginationPageLinkFocusBackgroundColor = "#9DFF40";
export const todoPaginationPageLinkFocusColor = "#000000";
export const todoPaginationPageLinkHoverTextDecoration = "underline";
export const todoPaginationPageLinkHoverBackgroundColor = "transparent";
export const todoPaginationPageLinkHoverBorderColor = "transparent";
export const todoPaginationPageLinkHoverColor = "#39051F";
export const todoPaginationPageLinkTextDecoration = "none";
export const todoPaginationPageLinkActiveTextDecoration = "underline";
export const todoPaginationPageLinkActiveBackgroundColor = "transparent";
export const todoPaginationPageLinkActiveBorderColor = "transparent";
export const todoPaginationPageLinkActiveColor = "#55072E";
export const todoPaginationPageLinkBackgroundColor = "transparent";
export const todoPaginationPageLinkBorderColor = "transparent";
export const todoPaginationPageLinkColor = "#850B48";
export const todoPaginationPageLinkMinBlockSize = "48px";
export const todoPaginationPageLinkMinInlineSize = "48px";
export const todoPaginationFontFamily = "'Fira Sans', Helvetica, Arial, sans-serif";
export const todoPaginationLineHeight = "150%";
export const todoPaginationFontSize = "18px";
export const todoPaginationEllipsesPaddingBlockEnd = "12px";
export const todoPaginationEllipsesPaddingBlockStart = "12px";
export const todoPaginationEllipsesPaddingInlineEnd = "4px";
export const todoPaginationEllipsesPaddingInlineStart = "4px";
export const todoPaginationEllipsesColor = "#787D87";
export const todoPaginationEllipsesFontWeight = 400;
export const todoPaginationDescriptionColor = "#122529";
export const todoPaginationDescriptionFontWeight = 400;
export const todoModalDialogBorderRadius = "16px";
export const todoModalDialogBoxShadow = {"x":0,"y":16,"blur":48,"spread":0,"color":"#E4E5E7","type":"dropShadow"};
export const todoModalDialogHeaderPaddingBlockStart = "8px";
export const todoModalDialogHeaderPaddingInlineStart = "16px";
export const todoModalDialogHeaderPaddingInlineEnd = "8px";
export const todoModalDialogHeaderPaddingBlockEnd = "8px";
export const todoModalDialogHeaderBackgroundColor = "transparent";
export const todoModalDialogHeaderLabelColor = "#122529";
export const todoModalDialogHeaderLabelFontFamily = "'Fira Sans', Helvetica, Arial, sans-serif";
export const todoModalDialogHeaderLabelFontWeight = 600;
export const todoModalDialogHeaderLabelFontSize = "18px";
export const todoModalDialogHeaderLabelLineHeight = "150%";
export const todoModalDialogHeaderBorderColor = "#122529";
export const todoModalDialogHeaderColumnGap = "8px";
export const todoModalDialogHeaderBorderWidth = "1px";
export const todoModalDialogBorderColor = "transparent";
export const todoModalDialogBackgroundColor = "#FFFFFF";
export const todoModalDialogColor = "#122529";
export const todoModalDialogFooterPaddingBlockStart = "16px";
export const todoModalDialogFooterPaddingInlineStart = "16px";
export const todoModalDialogFooterPaddingInlineEnd = "16px";
export const todoModalDialogFooterPaddingBlockEnd = "16px";
export const todoModalDialogFooterBackgroundColor = "transparent";
export const todoModalDialogFooterBorderColor = "transparent";
export const todoModalDialogFooterBorderWidth = "1px";
export const todoModalDialogContentPaddingBlockStart = "24px";
export const todoModalDialogContentPaddingBlockEnd = "24px";
export const todoModalDialogContentPaddingInline = "16px";
export const todoModalDialogBorderWidth = "1px";
export const todoIconOnlyButtonPaddingBlockEnd = "12px";
export const todoIconOnlyButtonPaddingBlockStart = "12px";
export const todoIconOnlyButtonPaddingInlineEnd = "12px";
export const todoIconOnlyButtonPaddingInlineStart = "12px";
export const todoFormFieldRadioOptionColumnGap = "8px";
export const todoFormFieldOptionLabelColor = "#1F4047";
export const todoFormFieldOptionLabelFontSize = "18px";
export const todoFormFieldOptionLabelLineHeight = "150%";
export const todoFormFieldOptionLabelFontFamily = "'Fira Sans', Helvetica, Arial, sans-serif";
export const todoFormFieldOptionLabelFontWeight = 400;
export const todoFormFieldOptionLabelDisabledColor = "#60646C";
export const todoFormFieldLabelColor = "#122529";
export const todoFormFieldLabelFontFamily = "'Fira Sans', Helvetica, Arial, sans-serif";
export const todoFormFieldLabelFontSize = "18px";
export const todoFormFieldLabelFontWeight = 600;
export const todoFormFieldLabelLineHeight = "150%";
export const todoFormFieldErrorMessageIconSize = "24px";
export const todoFormFieldErrorMessageIconMarginInlineEnd = "8px";
export const todoFormFieldCheckboxOptionColumnGap = "8px";
export const todoDrawerHeaderBorderWidth = "1px";
export const todoDrawerHeaderLabelColor = "#122529";
export const todoDrawerHeaderPaddingBlockEnd = "8px";
export const todoDrawerHeaderPaddingBlockStart = "8px";
export const todoDrawerHeaderPaddingInlineStart = "16px";
export const todoDrawerHeaderPaddingInlineEnd = "8px";
export const todoDrawerHeaderColumnGap = "8px";
export const todoDrawerHeaderBorderColor = "#122529";
export const todoDrawerHeaderBackgroundColor = "transparent";
export const todoDrawerBorderRadius = "0px";
export const todoDrawerFooterBorderWidth = "1px";
export const todoDrawerFooterPaddingBlockEnd = "16px";
export const todoDrawerFooterPaddingBlockStart = "16px";
export const todoDrawerFooterPaddingInlineStart = "16px";
export const todoDrawerFooterPaddingInlineEnd = "16px";
export const todoDrawerFooterBorderColor = "#122529";
export const todoDrawerFooterBackgroundColor = "transparent";
export const todoDrawerBorderWidth = "1px";
export const todoDrawerContentPaddingBlockEnd = "24px";
export const todoDrawerContentPaddingBlockStart = "24px";
export const todoDrawerContentPaddingInline = "16px";
export const todoDrawerBackgroundColor = "#FFFFFF";
export const todoDrawerBorderColor = "#122529";
export const todoDrawerColor = "#122529";
export const todoDrawerBoxShadow = {"x":0,"y":16,"blur":48,"spread":0,"color":"#E4E5E7","type":"dropShadow"};
export const todoDataListFontFamily = "'Fira Sans', Helvetica, Arial, sans-serif";
export const todoDataListItemValueFontWeight = 400;
export const todoDataListItemValueRowPaddingBlockEnd = "12px";
export const todoDataListItemValueRowPaddingBlockStart = "12px";
export const todoDataListItemValueRowPaddingInlineEnd = "0px";
export const todoDataListItemValueRowPaddingInlineStart = "0px";
export const todoDataListItemValuePaddingBlockEnd = "12px";
export const todoDataListItemValuePaddingBlockStart = "4px";
export const todoDataListItemValuePaddingInlineEnd = "0px";
export const todoDataListItemValuePaddingInlineStart = "0px";
export const todoDataListItemKeyFontWeight = 600;
export const todoDataListItemKeyRowPaddingBlockEnd = "12px";
export const todoDataListItemKeyRowPaddingBlockStart = "12px";
export const todoDataListItemKeyRowPaddingInlineEnd = "16px";
export const todoDataListItemKeyRowPaddingInlineStart = "0px";
export const todoDataListItemKeyPaddingBlockEnd = "4px";
export const todoDataListItemKeyPaddingBlockStart = "12px";
export const todoDataListItemKeyPaddingInlineEnd = "0px";
export const todoDataListItemKeyPaddingInlineStart = "0px";
export const todoDataListItemBorderWidth = "1px";
export const todoDataListItemBorderColor = "#122529";
export const todoDataListItemColor = "#122529";
export const todoDataListFontSize = "18px";
export const todoDataListLineHeight = "150%";
export const todoDataListItemActionRowPaddingBlockEnd = "12px";
export const todoDataListItemActionRowPaddingBlockStart = "12px";
export const todoDataListItemActionRowPaddingInlineEnd = "0px";
export const todoDataListItemActionRowPaddingInlineStart = "16px";
export const todoDataListItemActionPaddingBlockEnd = "12px";
export const todoDataListItemActionPaddingBlockStart = "12px";
export const todoDataListItemActionPaddingInlineEnd = "16px";
export const todoDataListItemActionPaddingInlineStart = "0px";
export const todoCheckboxGroupPaddingBlockEnd = "8px";
export const todoCheckboxGroupPaddingBlockStart = "8px";
export const todoCheckboxGroupRowGap = "16px";
export const todoBreadcrumbLineHeight = "150%";
export const todoBreadcrumbFontSize = "18px";
export const todoBreadcrumbLinkIconSize = "24px";
export const todoBreadcrumbLinkIconMarginInline = "4px";
export const todoBreadcrumbLinkPaddingBlockEnd = "12px";
export const todoBreadcrumbLinkPaddingBlockStart = "12px";
export const todoBreadcrumbLinkPaddingInlineEnd = "0px";
export const todoBreadcrumbLinkPaddingInlineStart = "0px";
export const todoBreadcrumbLinkColor = "#850B48";
export const todoBreadcrumbLinkActiveColor = "#55072E";
export const todoBreadcrumbLinkActiveTextDecoration = "none";
export const todoBreadcrumbLinkFocusBackgroundColor = "#9DFF40";
export const todoBreadcrumbLinkFocusColor = "#000000";
export const todoBreadcrumbLinkFocusTextDecoration = "none";
export const todoBreadcrumbLinkHoverColor = "#39051F";
export const todoBreadcrumbLinkHoverTextDecoration = "none";
export const todoBreadcrumbLinkCurrentColor = "#787D87";
export const todoBreadcrumbLinkTextDecoration = "underline";
export const todoBreadcrumbDividerSize = "16px";
export const todoBreadcrumbDividerColor = "#787D87";
export const todoBreadcrumbMarginInline = "8px";
export const todoBreadcrumbFontFamily = "'Fira Sans', Helvetica, Arial, sans-serif";
export const todoBreadcrumbFontWeight = 400;
export const todoBackdropBackgroundColor = "#E4E5E7";
export const todoBackdropColor = "#FFFFFF";
export const todoBackdropOpacity = 0.2;
export const todoBackdropReducedTransparencyOpacity = 0.98;
export const todoBackdropZIndex = "auto";
export const todoBackdropFadeInAnimationDuration = "0.4s";
export const todoAvatarBackgroundColor = "#850B48";
export const todoAvatarBorderRadius = "999px";
export const todoAvatarSize = "48px";
export const todoAvatarIconIconSize = "24px";
export const todoAvatarBorderColor = "transparent";
export const todoAvatarColor = "#FFFFFF";
export const todoAvatarTextFontFamily = "'Fira Sans', Helvetica, Arial, sans-serif";
export const todoAvatarTextFontWeight = 600;
export const todoAvatarTextLineHeight = "150%";
export const todoAvatarTextTextTransform = "uppercase";
export const todoAvatarTextFontSize = "18px";
export const todoAvatarBorderWidth = "1px";
export const todoAvatarHoverBackgroundColor = "#39051F";
export const todoAvatarHoverBorderColor = "transparent";
export const todoAvatarHoverColor = "#FFFFFF";
export const denhaagSidenavLinkLineHeight = "150%";
export const denhaagSidenavLinkIconSize = "24px";
export const denhaagSidenavLinkIconMarginInline = "8px";
export const denhaagSidenavLinkFontSize = "18px";
export const denhaagSidenavLinkFontFamily = "'Fira Sans', Helvetica, Arial, sans-serif";
export const denhaagSidenavLinkFontWeight = 400;
export const denhaagSidenavLinkCurrentFontWeight = 600;
export const denhaagSidenavLinkCurrentColor = "#122529";
export const denhaagSidenavLinkColor = "#850B48";
export const denhaagSidenavLinkActiveColor = "#55072E";
export const denhaagSidenavLinkActiveTextDecoration = "underline";
export const denhaagSidenavLinkFocusBackgroundColor = "#9DFF40";
export const denhaagSidenavLinkFocusColor = "#000000";
export const denhaagSidenavLinkFocusTextDecoration = "none";
export const denhaagSidenavLinkHoverColor = "#39051F";
export const denhaagSidenavLinkHoverTextDecoration = "underline";
export const denhaagSidenavLinkPaddingBlockStart = "12px";
export const denhaagSidenavLinkPaddingBlockEnd = "12px";
export const denhaagSidenavLinkTextDecoration = "none";
export const framelessRichTextMaxInlineSize = "800px";
export const framelessTypographyFontSizeSm = "16px";
export const framelessTypographyFontSizeMd = "18px";
export const framelessTypographyFontSizeLg = "20px";
export const framelessTypographyFontSizeXl = "24px";
export const framelessTypographyFontSize2xl = "32px";
export const framelessTypographyFontSize3xl = "40px";
export const framelessTypographyFontSize4xl = "48px";
export const framelessTypographyLineHeightXs = "100%";
export const framelessTypographyLineHeightSm = "125%";
export const framelessTypographyLineHeightMd = "150%";
export const framelessTypographyLineHeightLg = "200%";
export const framelessTypographyFontWeightNormal = 400;
export const framelessTypographyFontWeightBold = 600;
export const framelessTypographyFontFamilyPrimary = "'Fira Sans', Helvetica, Arial, sans-serif";
export const framelessColorWhite = "#FFFFFF";
export const framelessColorBlack = "#000000";
export const framelessColorGray10 = "#1E1F21";
export const framelessColorGray20 = "#303236";
export const framelessColorGray30 = "#484B51";
export const framelessColorGray40 = "#60646C";
export const framelessColorGray50 = "#787D87";
export const framelessColorGray60 = "#93979F";
export const framelessColorGray70 = "#AEB1B7";
export const framelessColorGray80 = "#C9CBCF";
export const framelessColorGray90 = "#E4E5E7";
export const framelessColorPetrol10 = "#122529";
export const framelessColorPetrol20 = "#1F4047";
export const framelessColorPetrol30 = "#2c5a63";
export const framelessColorPetrol40 = "#3C8790";
export const framelessColorPetrol50 = "#4BA7B2";
export const framelessColorPetrol60 = "#5EC1CC";
export const framelessColorPetrol70 = "#7FD8E6";
export const framelessColorPetrol80 = "#AAE5EE";
export const framelessColorPetrol90 = "#D4F2F7";
export const framelessColorPetrol95 = "#f2fbfd";
export const framelessColorSulkingroom60 = "#9C817D";
export const framelessColorRosewood50 = "#bd4271";
export const framelessColorRosewood60 = "#CA688E";
export const framelessColorRosewood70 = "#d78eaa";
export const framelessColorRosewood80 = "#E4B4C6";
export const framelessColorRosewood90 = "#F2D9E3";
export const framelessColorRosewood100 = "#FFFAFC";
export const framelessColorMaroon10 = "#39051F";
export const framelessColorMaroon20 = "#55072E";
export const framelessColorMaroon30 = "#850B48";
export const framelessColorMaroon40 = "#B30F61";
export const framelessColorGreen10 = "#082F13";
export const framelessColorGreen20 = "#0D4A1E";
export const framelessColorGreen30 = "#16822D";
export const framelessColorGreen50 = "#40C73E";
export const framelessColorGreen70 = "#83DC89";
export const framelessColorGreen90 = "#DDF8E4";
export const framelessColorRed10 = "#400000";
export const framelessColorRed30 = "#920707";
export const framelessColorRed40 = "#BA1212";
export const framelessColorRed50 = "#E01616";
export const framelessColorRed70 = "#F87373";
export const framelessColorRed90 = "#FDECEC";
export const framelessColorOrange20 = "#764504";
export const framelessColorOrange30 = "#A35F05";
export const framelessColorOrange40 = "#D57B07";
export const framelessColorOrange50 = "#F89009";
export const framelessColorOrange70 = "#FABC6B";
export const framelessColorOrange90 = "#FDEDD8";
export const framelessColorHighlighterGreen = "#9DFF40";
export const framelessSpaceInlineFlea = "2px";
export const framelessSpaceInlineAnt = "4px";
export const framelessSpaceInlineBeetle = "8px";
export const framelessSpaceInlineSnail = "12px";
export const framelessSpaceInlineMouse = "16px";
export const framelessSpaceInlineRat = "20px";
export const framelessSpaceInlineRabbit = "24px";
export const framelessSpaceInlineCat = "28px";
export const framelessSpaceInlineDog = "32px";
export const framelessSpaceInlinePig = "48px";
export const framelessSpaceBlockFlea = "2px";
export const framelessSpaceBlockAnt = "4px";
export const framelessSpaceBlockBeetle = "8px";
export const framelessSpaceBlockSnail = "12px";
export const framelessSpaceBlockMouse = "16px";
export const framelessSpaceBlockRat = "20px";
export const framelessSpaceBlockRabbit = "24px";
export const framelessSpaceBlockCat = "32px";
export const framelessSpaceBlockDog = "48px";
export const framelessSpaceBlockPig = "64px";
export const framelessSpaceTextFlea = "1px";
export const framelessSpaceTextAnt = "2px";
export const framelessSpaceTextBeetle = "4px";
export const framelessSpaceTextSnail = "6px";
export const framelessSpaceTextMouse = "8px";
export const framelessSpaceTextRat = "12px";
export const framelessSpaceTextRabbit = "14px";
export const framelessSpaceTextCat = "16px";
export const framelessSpaceTextDog = "24px";
export const framelessSpaceColumnFlea = "1px";
export const framelessSpaceColumnAnt = "2px";
export const framelessSpaceColumnBeetle = "4px";
export const framelessSpaceColumnSnail = "8px";
export const framelessSpaceColumnMouse = "12px";
export const framelessSpaceColumnRat = "16px";
export const framelessSpaceColumnRabbit = "20px";
export const framelessSpaceColumnCat = "24px";
export const framelessSpaceColumnDog = "28px";
export const framelessSpaceColumnPig = "32px";
export const framelessSpaceColumnTiger = "48px";
export const framelessSpaceColumnHorse = "64px";
export const framelessSpaceColumnElephant = "96px";
export const framelessSpaceColumnGiraffe = "160px";
export const framelessSpaceRowFlea = "1px";
export const framelessSpaceRowAnt = "2px";
export const framelessSpaceRowBeetle = "4px";
export const framelessSpaceRowSnail = "8px";
export const framelessSpaceRowMouse = "12px";
export const framelessSpaceRowRat = "16px";
export const framelessSpaceRowRabbit = "20px";
export const framelessSpaceRowCat = "24px";
export const framelessSpaceRowDog = "28px";
export const framelessSpaceRowPig = "32px";
export const framelessSpaceRowTiger = "48px";
export const framelessSpaceRowHorse = "64px";
export const framelessSpaceRowElephant = "96px";
export const framelessSpaceRowGiraffe = "160px";
export const framelessBorderRadiusSm = "4px";
export const framelessBorderRadiusMd = "8px";
export const framelessBorderRadiusLg = "16px";
export const framelessBorderRadiusRound = "999px";
export const framelessBorderWidthSm = "1px";
export const framelessBorderWidthMd = "2px";
export const framelessBorderWidthLg = "4px";
export const framelessSize5xs = "2px";
export const framelessSize4xs = "4px";
export const framelessSize3xs = "8px";
export const framelessSize2xs = "16px";
export const framelessSizeXs = "24px";
export const framelessSizeSm = "32px";
export const framelessSizeMd = "48px";
export const framelessSizeLg = "64px";
export const framelessSizeXl = "96px";
export const framelessSize2xl = "160px";
export const framelessSizeIconSm = "16px";
export const framelessSizeIconMd = "24px";
export const framelessSizeIconLg = "24px";
export const framelessSizeIconXl = "24px";
export const framelessSizeIcon2xl = "32px";
export const framelessSizeIcon3xl = "40px";
export const framelessSizeIcon4xl = "48px";
export const framelessSizeMaxWidthSm = "100px";
export const framelessSizeMaxWidthMd = "200px";
export const framelessSizeMaxWidthLg = "500px";
export const framelessBoxShadowSm = {"x":0,"y":2,"blur":6,"spread":0,"color":"#E4E5E7","type":"dropShadow"};
export const framelessBoxShadowMd = {"x":0,"y":8,"blur":16,"spread":0,"color":"#E4E5E7","type":"dropShadow"};
export const framelessBoxShadowLg = {"x":0,"y":16,"blur":48,"spread":0,"color":"#E4E5E7","type":"dropShadow"};
export const framelessLogoTextFontFamily = "'Fira Sans', Helvetica, Arial, sans-serif";
export const framelessLogoTextFontSize = "48px";
export const framelessLogoTextFontWeight = 400;
export const framelessLogoTextSmallFontSize = "32px";
export const framelessLogoWrapperInlineSize = "12px";
export const framelessLogoWrapperPaddingBlock = "12px";
export const framelessLogoWrapperPaddingInline = "12px";
export const framelessCardHeadingFontSize = "24px";
export const framelessCardHeadingMarginBlockEnd = "20px";
export const framelessCardFontFontFamily = "'Fira Sans', Helvetica, Arial, sans-serif";
export const framelessCardColorBackgroundColor = "#f2fbfd";
export const framelessCardParagraphFontSize = "16px";
export const framelessCardParagraphMarginBlockEnd = "20px";
export const framelessCardBorderBorderRadius = "8px";
export const framelessCardSpacingMaxWidth = "500px";
export const framelessCardSpacingPaddingBlockStart = "12px";
export const framelessCardSpacingPaddingBlockEnd = "12px";
export const framelessCardSpacingPaddingInlineStart = "24px";
export const framelessCardSpacingPaddingInlineEnd = "24px";
export const framelessCardGroupGap = "24px";
export const framelessCardLineHeightLineHeight = "125%";
export const tokenSetOrder0 = "brand";
export const tokenSetOrder1 = "common";
export const tokenSetOrder2 = "components/accordion";
export const tokenSetOrder3 = "components/alert";
export const tokenSetOrder4 = "components/avatar";
export const tokenSetOrder5 = "components/backdrop";
export const tokenSetOrder6 = "components/blockquote";
export const tokenSetOrder7 = "components/breadcrumb";
export const tokenSetOrder8 = "components/button";
export const tokenSetOrder9 = "components/button-group";
export const tokenSetOrder10 = "components/checkbox";
export const tokenSetOrder11 = "components/checkbox-group";
export const tokenSetOrder12 = "components/counter-badge";
export const tokenSetOrder13 = "components/data-list";
export const tokenSetOrder14 = "components/drawer";
export const tokenSetOrder15 = "components/form-field";
export const tokenSetOrder16 = "components/form-field-checkbox-option";
export const tokenSetOrder17 = "components/form-field-description";
export const tokenSetOrder18 = "components/form-field-error-message";
export const tokenSetOrder19 = "components/form-field-label";
export const tokenSetOrder20 = "components/form-field-option-label";
export const tokenSetOrder21 = "components/form-field-radio-option";
export const tokenSetOrder22 = "components/heading";
export const tokenSetOrder23 = "components/icon-only-button";
export const tokenSetOrder24 = "components/link";
export const tokenSetOrder25 = "components/modal-dialog";
export const tokenSetOrder26 = "components/ordered-list";
export const tokenSetOrder27 = "components/pagination";
export const tokenSetOrder28 = "components/paragraph";
export const tokenSetOrder29 = "components/radio";
export const tokenSetOrder30 = "components/radio-group";
export const tokenSetOrder31 = "components/select";
export const tokenSetOrder32 = "components/separator";
export const tokenSetOrder33 = "components/side-nav";
export const tokenSetOrder34 = "components/skip-link";
export const tokenSetOrder35 = "components/status-badge";
export const tokenSetOrder36 = "components/table";
export const tokenSetOrder37 = "components/task-list";
export const tokenSetOrder38 = "components/textarea";
export const tokenSetOrder39 = "components/text-input";
export const tokenSetOrder40 = "components/toolbar-button";
export const tokenSetOrder41 = "components/unordered-list";
